bitnami/harbor/templates/registry/registry-ctl-cm-envvars.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
Shouldn't we also disable this configmap if there are no configurations enabled? Since the tracing feature is optional, it would cause empty configmaps to be created where it is not enabled.

Reply:
Done,

This works fine for now where we only have the tracing variables in the config map but it'll be more difficult if the condition for creating it becomes more complex.